cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/ApacheKafkaTopicService.java

```diff
@@ -96,7 +96,7 @@ private Map<String, TopicDescription> loadTopicDescriptionsMap(AdminClient clien
 
   public synchronized ApiResponse createTopic(ClusterTopicRequest clusterTopicRequest)
       throws Exception {
-    log.info("createTopic {}", clusterTopicRequest);
+    log.info("createTopic {}", clusterTopicRequest.toString());
     AdminClient client =
```

Comment:
You shouldnt need to specify toString() with the logger

Reply:
Earlier it was printing object id, tried adding toString, but realised its better with toString annotation, and forgot to remove this. toString annotation works.